Gruntfile.js

```diff
@@ -67,7 +71,13 @@ module.exports = function(grunt) {
           declaration: true,
           module: 'commonjs'
         },
-        src: ['src/tslint.ts'],
+        src: [
+          'typings/*.d.ts',
+          'src/language/walker/syntaxWalker.ts',
```

Comment:
yes, it's for ordering. I'll investigate if there's a way to avoid this by using external modules.